cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,207 @@
++
+** The drop-down for the AWS billing account on the screenshot above should be set to the AWS account that accepted the private offer, providing the purchased quota is intended to be used by the cluster that is being created. If different AWS accounts are selected in the infrastructure and billing _roles_, the blue informative note visible in Figure 10 is shown.
+
+////
+//Commenting out for now, but we can examine how to incorporate the example scenario 
+
+== Example scenario
+
+* John is developer who briefly tested {hcp-title} by activating the service using their AWS account 123412341234, then deleted the cluster after the trial was completed.
+* John is responsible for the company's main AWS account 111111111111 and manages their AWS organization with several linked AWS accounts.
+* John accepts a private offer for {hcp-title} as described in _Accepting a private offer_.
+* John connects the AWS account 111111111111, that was used for accepting the private offer, with their Red{nbsp}Hat account right after accepting the private offer as described in _Connecting AWS and Red{nbsp}Hat accounts_.
+* Anne wants to deploy a new ROSA cluster. Normally, they use their AWS account 123412341234 which is linked to the master AWS account 111111111111 for all cloud infrastructure needs.
+* John makes sure that Anne Red{nbsp}Hat user is in the same Red{nbsp}Hat organization as John’s Red{nbsp}Hat user. John is the Red{nbsp}Hat organization administrator and can do that here.
+* Anne visits the Red{nbsp}Hat Hybrid Cloud Console and starts a new cluster deployment.
+* During the initial steps, Anne makes sure to select 111111111111 as the AWS billing account, while using thier own AWS account 123412341234 as the AWS infrastructure account.
+* Note that Anne is also able to select their own AWS account 123412341234 as the billing account when creating a cluster because they previously activated the ROSA with HCP public offer. However, that would result in custom private offer pricing not being applied to this cluster. Therefore it is important for John to communicate the information about which AWS billing account is to be used by employees when creating a new cluster in order to benefit from a private offer that was accepted before.
+////
+
+== Troubleshooting
+
+The most frequent issues associated with private offer acceptance and Red{nbsp}Hat account linking.
+
+=== Accessing a private offer using a different AWS account
+
+* If you try accessing the private offer when logged in under AWS account ID that is not defined in the offer, and see the message shown in Figure 11, then verify that you are logged in as the desired AWS billing account. 
++
+.HTTP 404 error when using the private offer URL
++
+image::rosa-http-404-error-when-using-the-private-offer-url.png[]
++
+** Contact the seller if you need the private offer to be extended to another AWS account.
```

Comment:
Should this be a level 2? I would think that level 1 or no level with a `+` after the image could work.